src/main/java/io/mosip/mimoto/service/impl/CredentialServiceImpl.java

```diff
@@ -135,7 +139,6 @@ public ByteArrayInputStream downloadCredentialAsPDF(String issuerId, String cred
     public VCCredentialResponse downloadCredential(String credentialEndpoint, VCCredentialRequest vcCredentialRequest, String accessToken) throws InvalidCredentialResourceException {
         VCCredentialResponse vcCredentialResponse = restApiClient.postApi(credentialEndpoint, MediaType.APPLICATION_JSON,
                 vcCredentialRequest, VCCredentialResponse.class, accessToken);
-        log.debug("VC Credential Response is -> " + vcCredentialResponse);
```

Comment:
I think by mistake this log is removed. @kamlesh012

Reply:
Readded.